src/lib/Preview.js

```diff
@@ -513,25 +513,27 @@ class Preview extends EventEmitter {
      * Downloads the file being previewed.
      *
      * @public
-     * @return {void}
+     * @return {Promise} download promise
```

Comment:
This is a public API for the preview instance, right? Is this considered a breaking change?

Reply:
I am actually gonna remove the return promise. Only downside is that this function is not really being tested properly, specially for errors.